.github/workflows/build_docs.yml

```diff
@@ -46,5 +46,6 @@ jobs:
       run: |
         git config --global user.name 'LF1 Bot'
         git config --global user.email 'lf1_bot@users.noreply.github.com'
+        git add .
```

Comment:
To be a little safer could we do `git add docs/`? I'm afraid with the current call we could accidentally files we don't mean to.

Reply:
yes, makes sense, I'll change